packages/core/src/crypto/WalletKeyPair.ts

```diff
@@ -1,15 +1,14 @@
 import type { Wallet } from '..'
```

Comment:
No import from '..' (not your in your PR though :))

Reply:
> No import from '..' (not your in your PR though :))

The classic one!